lib/common.py

```diff
@@ -16,6 +17,17 @@ def vm_image(vm_key):
         url = DEF_VM_URL + url
     return url
 
+def objects_name_prefix():
+    try:
+        from data import OBJECTS_NAME_PREFIX
+        name_prefix = OBJECTS_NAME_PREFIX
+    except ImportError:
+        name_prefix = f"[{os.getlogin()}]"
```

Comment:
According to https://docs.python.org/3/library/os.html#os.getlogin `getpass.getuser()` should be preferred over `os.get_login()`.